frontend/sw360-portlet/src/main/java/org/eclipse/sw360/portal/portlets/components/ComponentPortlet.java

```diff
@@ -1840,10 +1840,37 @@ private void prepareReleaseDetailView(RenderRequest request, RenderResponse resp
 
             component = client.getAccessibleComponentById(id, user);
             request.setAttribute(COMPONENT, component);
-            for (int i=0; i<component.releases.size(); i++) {
-                Collections.sort(component.releases, (release1, release2) -> release1.getVersion() .compareTo(release2.getVersion()) );
-            }
-            Collections.reverse(component.releases);
+            Comparator<Release> releaseVersion = new Comparator<Release>() {
```

Comment:
Calling the variable `releaseVersion` is confusing. Can you please add `Comparator` to make it differentiable?
```suggestion
            Comparator<Release> releaseVersionComparator = new Comparator<Release>() {
```

Reply:
done.